addons/account_asset/static/src/js/account_asset.js

```diff
@@ -0,0 +1,22 @@
+openerp.account_asset = function (instance) {
```

Comment:
can not use toggle_button widget ? (https://github.com/odoo/odoo/blob/master/addons/web/static/src/js/view_list.js#L2438)